ci/Dockerfile.micro

```diff
@@ -31,7 +31,7 @@ RUN ln -s /usr/bin/clang-format-12 /usr/bin/clang-format
 # integration.
 RUN pip install yapf
 
-RUN pip install six
+RUN pip install six Pillow
```

Comment:
nit: line 32 and 37 both does pip with one package on one line. Just wondering whether it makes sense to have one package at one line as a rule of thumb.  This is really NIT and totally up to you.

Reply:
done. also added a comment about why Pillow was added